.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Go
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ '1.21.x' ]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v4
```

Comment:
<img width="1073" alt="image" src="https://github.com/VanuPhantom/skrive/assets/144150574/1c27786a-f62a-4fc6-a262-23ae61d79c83">

This might be an outdated action.

Reply:
Fixed, see https://github.com/freyacodes/skrive/actions/runs/7846925357/job/21414720419